Tests/ApodiniDeployTests/LambdaDeploymentProviderTests.swift

```diff
@@ -19,15 +20,13 @@ import SotoApiGatewayV2
 
 
 class LambdaDeploymentProviderTests: ApodiniDeployTestCase {
-    private var task: Task! // swiftlint:disable:this implicitly_unwrapped_optional
+    private var task: ChildProcess! // swiftlint:disable:this implicitly_unwrapped_optional
     private var taskStdioObserverToken: AnyObject?
     
     
     func testLambdaDeploymentProvider() throws { // swiftlint:disable:this function_body_length cyclomatic_complexity
-        guard Self.shouldRunDeploymentProviderTests else {
-            print("Skipping test case '\(#function)'.")
-            return
-        }
+        throw XCTSkip() // Remove this once the lambda DP is fixed
+        //try XCTSkipUnless(Self.shouldRunDeploymentProviderTests)
```

Comment:
Nope. As mentioned in the PR description, the lambda DP is currently broken, hence we have to disable the test.
(This will be fixed in a later PR, since it's not that trivial, but @PSchmiedmayer agreed that it'd be ok to commit the temporarily not-working DP)